src/quicktype-core/language/Python.ts

```diff
@@ -417,6 +419,10 @@ export class PythonRenderer extends ConvenienceRenderer {
                 if (t.getProperties().size === 0) {
                     this.emitLine("pass");
                 } else {
+                    const sort = (v: ClassProperty) => (
+                        Boolean(v.type instanceof UnionType && Boolean(nullableFromUnion(v.type))) ? 1 : 0
```

Comment:
Who do you need this outer `Boolean`?

Reply:
tslint warnings for `null is not boolean` or something like that